frb_spectral_search/acf_func.py

```diff
@@ -196,7 +197,7 @@ def gauss_smooth_divide(spec , wid):
     smoothdivided_spec = spec.copy()
     smoothdivided_spec[spec != 0 ] = spec_no0/smooth #changed - to / just to try, no work,
     smoothdivided_spec[spec == 0 ] = np.nan
-    return np.array(smoothdivided_spec[150:-149]) 
```

Comment:
I think the `[150:-149]` construction is OK but I don't like that it's hard-coded. What if you change the width of the smoothing spline? If you pass an argument like `width` this could be set to `[int(width//2):-int(width//2)]` throughout.